### ISSUE TITLE
Is this a winboard fork?

### ISSUE THREAD

**Issue by gekkehenker** [fire-lizard/QBoard#1]
See title.

Kinda interested in this.

**Comment by fire-lizard** [comment]
Yes, I have tried to make a Winboard fork since it was not updated for a while
________________________________
From: gekkehenker ***@***.***>
Sent: Monday, April 5, 2021 11:52 PM
To: fire-lizard/QBoard ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [fire-lizard/QBoard] Is this a winboard fork? (#1)


See title.

Kinda interested in this.
